setup.py

```diff
@@ -62,8 +62,6 @@ def package_files(directory, strip_leading):
           ],
           'pc': [
               'matplotlib',
-              'imgaug',
```

Comment:
Why did you remove these from `setup.py` ? 

I think these dependencies are still relevant.